config/runtime.exs

```diff
@@ -5,6 +5,9 @@ if config_env() == :prod do
     port: System.fetch_env!("HEXDOCS_PORT"),
     hexpm_url: System.fetch_env!("HEXDOCS_HEXPM_URL"),
     hexpm_secret: System.fetch_env!("HEXDOCS_HEXPM_SECRET"),
+    typesense_url: System.fetch_env!("TYPESENSE_URL"),
+    typesense_api_key: System.fetch_env!("TYPESENSE_API_KEY"),
```

Comment:
This would be the key for https://cloud.typesense.org/clusters/ent97o5sv4dzx2f0p cluster, it acn be downloaded from the dashboard.